cmd/janitor.go

```diff
@@ -52,6 +52,8 @@ Janitor can be used in several ways.
 		RunE: cli.NewHandler().Janitor.RunE,
 		Args: cli.NewHandler().Janitor.Args,
 	}
+	cmd.Flags().Int(cli.Limit, 10000, "Limits the number of records retrieved from database for deletion.")
+	cmd.Flags().Int(cli.BatchSize, 100, "Limits the number of records retrieved from database for deletion.")
```

Comment:
we might have to check if these numbers are valid, not 0 or something outrageous and give a proper response to the user?

Reply:
with a proper production-tier database I think this is a reasonable batch size (as it is in my company's case), but I think this may strongly depend on the database host, so maybe we should make them mandatory like --keep-if-younger?

Reply:
I think just set sane-defaults and keep the flags optional. We can then add some documentation to what is recommended for large databases.

You can then also do some checks to make sure whatever the user is passing won't cause issues, like passing a 0 or something too big here https://github.com/ory/hydra/blob/eddfa2d648ae94c5056d4568a34a922c8487f6be/cmd/cli/handler_janitor.go#L39

We will most likely require some tests for that, just to test if the error given is expected or not etc.
an example is to test if `--limit = 0` or if `batch-size=0` then expect some error.